EIPS/eip-6110.md

```diff
@@ -76,11 +76,35 @@ Beginning with the `FORK_BLOCK`, each deposit accumulated in the block **MUST**
 in the order they appear in the logs. To illustrate:
 
 ```python
-def parse_deposit_data(deposit_event_data) -> bytes[]:
-  """
-  Parses deposit data from DepositContract.DepositEvent data
-  """
-  pass
+def parse_deposit_data(deposit_event_data):
+    """
+    Parses raw emitted EVM log data to decode the request data
+    """
+
+
+    # These offsets points are data pointers which first encode the 
+    # size of the data (which is always 32 bytes)
+    # The data itself (of this size) is then appended after the size 32 bytes
+    PUBKEY_OFFSET = deposit_event_data[0:32]
+    WITHDRAWAL_CREDENTIALS_OFFSET = deposit_event_data[32:64]
+    AMOUNT_OFFSET = deposit_event_data[64:96]
+    SIGNATURE_OFFSET = deposit_event_data[96:128]
+    INDEX_OFFSET = deposit_event_data[128:160]
```

Comment:
What if any of these offsets or derived values is outside of the actual data length?

Reply:
Yes, good point, I will add in that if this is the case (the LOG is thus invalid) then it is thus also not a valid request and should be skipped

Reply:
This does have the complication that it's "system tx" level so will fail the block (as per Sepolia)

Which flows into the other discussion about what happens when a system tx fails

Reply:
In this case it meant the contract produced an invalid log data thus invalidating a block would be reasonable.